dotnet/src/IntegrationTests/Connectors/MistralAI/ChatCompletion/MistralAIChatCompletionTests.cs

```diff
@@ -130,12 +128,40 @@ public async Task ValidateGetChatMessageContentsHasToolCallsResponseAsync()
         {
             new ChatMessageContent(AuthorRole.User, "What is the weather like in Paris?")
         };
+        var executionSettings = new MistralAIPromptExecutionSettings { ToolCallBehavior = MistralAIToolCallBehavior.EnableKernelFunctions };
+        var response = await service.GetChatMessageContentsAsync(chatHistory, executionSettings, kernel);
+
+        // Assert
+        Assert.NotNull(response);
+        Assert.Single(response);
+        Assert.Equal("tool_calls", response[0].Metadata?["FinishReason"]);
+    }
+
+    [Fact] // (Skip = "This test is for manual verification.")
```

Comment:
Skip back this and others?

Reply:
Leaving as is while this is in a feature branch as I'm running these tests a lot. I will resolve this when I create the PR to merge to main.